tsconfig.json

```diff
@@ -1,9 +1,19 @@
 {
   "compilerOptions": {
+    "jsx": "react",
     "strict": true,
-    "target": "es5",
-    "lib": ["es5", "dom"],
-    "types": ["node", "jest"]
+    "esModuleInterop": true,
+    // "target": "esnext",
+    "lib": ["esnext", "dom"],
+    "types": ["node", "jest", "@testing-library/jest-dom"],
+    "allowJs": true,
+    "resolveJsonModule": true,
+    // "checkJs": true,
```

Comment:
?

Reply:
Commented out for discussion - I was finding that compiler was not happy with the options set (including validating JS files as TS). But open for discussion.